### PR TITLE
chore: add brave to early testers

### PR DESCRIPTION
**Pull request by galargh**
This is related to https://github.com/ipfs/kubo/issues/8576. @lidel who would we want to mention here?

### CODE REVIEW

File: docs/EARLY_TESTERS.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ We will ask early testers to participate at two points in the process:
 - [ ] Charity Engine (@rytiss, @tristanolive)
 - [ ] Fission (@bmann)
 - [ ] OrbitDB (@aphelionz)
+- [ ] Brave ()
```
Comment (lidel) [inline docs/EARLY_TESTERS.md:35]: Hi @cypt4!

Before each stable release, we are making at least one Release Candidate release,
allowing projects that build on Kubo to provide early feedback.

Would Brave be interested in getting pinged when RC release happens, (adding you here) 
or do you prefer to only use final stable releases?

**Comment by BigLep** [comment]
@galargh : did this get resolved?

**Comment by galargh** [comment]
> @galargh : did this get resolved?

Nope, this is the last activity we had on this - https://github.com/ipfs/kubo/pull/9487/#discussion_r1044392724

**Comment by lidel** [comment]
Not happening, Brave QA process was to always test the final release, they did not allocate time for RCs.

**Pull request closed without merging**